pyblish_ftrack/plugins/conform_ftrack_component_preview.py

```diff
@@ -0,0 +1,43 @@
+import pyblish.api
+import os
+import ftrack
+
+@pyblish.api.log
+class FtrackUploadPreview(pyblish.api.Conformer):
+    """Creates Preview component within supplied version, also makes this preview reviewable.
+
+    Expected data members:
+    'publishedFile' - path that will be saved as a component
+    'ftrackVersionID' - ID of a version where component should be created
```

Comment:
The thing about using `Arguments` specifically, is that we could use an existing docstring parser, like Sphinx to retrieve the components.

There is also `Attributes`, but I'm not sure it supports documenting the signature, datatype and whether it is optional or not.

Either way, it wouldn't appear anywhere but the plug-in source.